simcore/simulation/library/anchor.hpp

```diff
@@ -23,6 +23,8 @@ class Anchor : public Object {
   double velocity_;
   double max_velocity_, diffusion_;
   double k_off_;
+  double k_on_d_;
```

Comment:
Anchor does not need k_on_d_ if the binding rates are passed by reference, as mentioned in a previous comment.